src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/WebClientConfig.kt

```diff
@@ -34,37 +34,27 @@ class WebClientConfig(
       )
       .build()
 
-  @Value("\${bankHolidayRegister.endpoint.url}")
-  private lateinit var bankHolidayRegisterEndpointUrl: String
-
   @Bean("bankHolidayRegisterWebClient")
-  fun bankHolidayRegisterWebClient(): TimeoutHandlingWebClient = timeoutHandlingWebClient(bankHolidayRegisterEndpointUrl)
-
-  @Value("\${courtRegister.endpoint.url}")
-  private lateinit var courtRegisterEndpointUrl: String
+  fun bankHolidayRegisterWebClient(@Value("\${bankHolidayRegister.endpoint.url}") bankHolidayRegisterEndpointUrl: String):
+    TimeoutHandlingWebClient = timeoutHandlingWebClient(bankHolidayRegisterEndpointUrl)
```

Comment:
Cosmetic, but maybe drop the explicit return type (`: TimeoutHandlingWebClient`): the other equivalents below don't have it.